src/URLDetector.php

```diff
@@ -148,19 +156,27 @@ public static function detectURLs() : string {
 
         $unique_urls = array_unique( $url_queue );
 
+        $total_detected = (string) count( $unique_urls );
+
+        if ( ! $quiet ) {
+            WsLog::l(
+                "Detection complete. $total_detected URLs added to Crawl Queue."
+            );
+        }
+
+        return $unique_urls;
+    }
+
+    public static function enqueueURLs() : string {
```

Comment:
Why does CrawlQueue always equal the DetectedURLs?

I envision the `status` command being an observation. You wan to see the current status of the system. When you're observing something, you don't want to change it - you just want to see how it is. If the act of observing (running "detect urls") actually changed something (by adding URLs to the queue) then it's no longer simply observing, it's changing the system. I'm envisioning `status` as being something that can be run without any side effects.

Reply:
Cool, so I think we can add a `status` commnd without altering anything else. Within any status check, we can check the `CrawlQueue` to get current detected URLs.

I still need some convincing on a scenario where you want to detect without enqueing, which is different than a status check, as it is actively doing something.

Maybe a `wp wp2static detect --dry-run` or `--skip-enqueue` would meet the scenario's needs?

Reply:
I think the use case here is that status can tell which step needs to be run. Without using the detect logic, status can tell you if you need to run crawl, post_process, or deploy, but it can't tell whether you need to run detect. This does seem useful for helping people learn the system or diagnose config problems.

A `detect --dry-run` sounds really nice though. Maybe we can add that sometime along with a `--verbose` flag.

Reply:
Cool, I'm convinced :D @bookwyrm - able to update the PR and I'll merge it in!